doc_source/modules-structure.md

```diff
@@ -7,7 +7,7 @@ A module consists of two main pieces:
 ## Creating the module template fragment<a name="modules-template-fragment"></a>
 
 The starting point for developing a module is the template fragment\. The template fragment is a file that contains the information that defines the resources for CloudFormation to provision during stack operations, including:
-+ A `[Resources](https://docs.aws.amazon.com/)` section that defines the resources to be provisioned\.
++ A `[Resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html)` section that defines the resources to be provisioned\.
```

Comment:
We should remove the back-ticks around the Resources link. The back-ticks prevent it from being linkable.

Reply:
Thanks for reviewing, I've removed the back-ticks

Reply:
Thank you! Running the test now. When they pass we should be able to merge this PR.